### ISSUE TITLE
Clarification on Training Datasets with Metric Ground-truth

### ISSUE THREAD

**Issue by Forrest-110** [naver/mast3r#11]
Hi,

Great work on your paper!

In Section 4.1 of your paper, you mention using 14 datasets for training, with metric ground-truth available for 10 of them. Could you please specify which datasets include this ground-truth? It would be of great help!

Thank you!

**Comment by yocabon** [comment]
Hi,
I think the list is:
ARKitScenes, Habitat, Niantic Map-Free, our internal unreleased dataset (uses lidar data), ScanNet++, TartanAir, UnrealStereo4K, VirtualKitti, Waymo (note: not used for the checkpoint we released), WildRgbd

**Comment by YongtaoGe** [comment]
Hi, @yocabon, will you release the preprocess scripts of these datasets with metric ground truth?

**Comment by yocabon** [comment]
ARKitScenes, Habitat, ScanNet++, Waymo, WildRGB-D are already available in dust3r, the preprocess script did not change. For the others, I cannot say.

**Comment by TwiceMao** [comment]
I hope this message finds you well. @yocabon  I have a query regarding your use of the MegaDepth dataset in your training set. It appears that the MegaDepth dataset lacks ground truth camera poses and depth information. Could you please clarify how you obtained the ground truth for the MegaDepth dataset used in the regression loss? Additionally, I am interested in understanding how you obtained the ground-truth correspondences  $\hat{M}$  for the matching objective.

**Comment by sowmya-mp** [comment]
Hi, the paper also reported that Static Scenes 3D and Blended MVS are used to train MASt3R. I was wondering why we aren't including these two in the 10 datasets listed above as they have indoor synthetic scenes with access to metric GT
